locale/pt-br/texts.po

```diff
@@ -1180,10 +1221,30 @@ msgstr "Processando"
 msgid "Voided"
 msgstr "Inválida"
 
-#: src/components/TxDetailsModal.js:60
+#: src/components/TxDetailsModal.js:101
 msgid "Description"
 msgstr "Descrição"
 
+#: src/components/TxDetailsModal.js:103
+msgid "Transaction ID"
+msgstr "ID da Transação"
+
+#: src/components/TxDetailsModal.js:104
+msgid "Blueprint Method"
+msgstr "Método Blueprint"
+
+#: src/components/TxDetailsModal.js:105
+msgid "Nano Contract ID"
+msgstr "ID do Nano Contract"
+
+#: src/components/TxDetailsModal.js:106
+msgid "Nano Contract Caller"
+msgstr "Acionador do Nano Contract"
```

Comment:
What would be the best person to discuss this translation with? Campana?

I don't like this translation, but I don't have anything better to suggest, so I'm going to approve this PR

Reply:
I don't like it either. While we wait for the best names from the other areas, I think we can call "Endereço que assinou o Nano Contract", even if it's big, I think it's better.

What do you think?

Reply:
@alexruzenhack 

At the moment, in our documentation about nano contracts, this field was named "caller", you can see in the diagram at:
https://beta-test.docs.hathor.network/explanations/features/nano-contracts/how-it-works#transactions

However, please note that this name was provided for two reasons:
- This is how Marcelo named it.
- It relates to who is indeed calling the contract.

That said, although this nomenclature is adequate for the documentation, it might not be suitable in terms of UI/UX. Because I don't know the overall terminology of this and also other features in our UIs, is difficult to suggest something at this moment.

Reply:
Hi, @BrunoCampana, thank you to add more light in this case. I got it!

Regarding the translation, do you do have a suggestion?

I found a better word for the translation so far: "Signatário"; see [definition](https://www.dicio.com.br/signatario/). What do you guys say about this one?